app/views/pregnancies/_call_log.html.erb

```diff
@@ -8,6 +8,10 @@
 			<th>Result</th>
 			<th>CM</th>
 		</tr>
+		<tbody>
```

Comment:
Can you revert changes to these four lines? Call logs are handled in the loop below this.